android_jni/avifandroidjni/src/main/jni/libavif_jni.cc

```diff
@@ -175,24 +188,39 @@ FUNC(jboolean, isAvifImage, jobject encoded, int length) {
   return avifPeekCompatibleFileType(&avif);
 }
 
+#define CHECK_EXCEPTION(ret) \
+  if (JniExceptionCheck(env)) return ret
```

Comment:
I saw your comment to Wan-Teh's question. I think this one could be wrapped in a do/while though

Reply:
Done.